plans/general.fmf

```diff
@@ -33,6 +33,7 @@ environment:
     SR_ARTIFACTS_URL: ""
     # new var name: SR_TFT_DEBUG: false
     LSR_TFT_DEBUG: false
+    SR_REPORT_ERRORS_URL: "https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/dd84e61663d7d1c7b9e9f4ad6f8b9d0a83997259/callback_plugins/lsr_report_errors.py"
```

Comment:
Please move this to test parameters in https://github.com/linux-system-roles/tft-tests/blob/main/tests/general/test.sh#L5 and https://github.com/linux-system-roles/tft-tests/blob/main/tests/mssql_ha/mssql_ha.sh